### DISCUSSION TITLE
Prusa connect

### DISCUSSION THREAD

**Discussion by Norbertocash** [community/community#146746] [New to GitHub]
I have two i3 MK3S+ printers connected to Prusa Connect via a Raspberry Pi3. Up until 15 days ago, they were working normally and even printing simultaneously, with addresses 192.168.0.126/1 and /2. One of the printers was down for about 10 days. Now I can't access this printer anymore because it's always OFFLINE. The printer panel shows the message Prusa Link OK and the correct IP. How do I put the printer online? Do I need to load the prusa_printer_seetings.ini file onto the printer? How do I load this file?

**Comment by tuves** [comment]
Hi @Norbertocash,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on this project, such as the [raspberry-prusa-connect's repository](https://github.com/LIII-XXII/raspberry-prusa-connect). It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
